base/dates/parse.jl

```diff
@@ -281,9 +281,14 @@ function Base.tryparse{T<:TimeType}(
     pos, len = start(str), endof(str)
     values, pos = tryparsenext_internal(T, str, pos, len, df, false)
     if isnull(values)
-        Nullable{T}()
+        return Nullable{T}()
     else
-        Nullable{T}(T(unsafe_get(values)...))
+        try
+            parsed = Nullable{T}(T(unsafe_get(values)...))
```

Comment:
Oh I guess that's an interesting reversal, where null would mean no-exception, things are fine, it is valid.